Web/Pages/CCGMember/Compare/Delta.cshtml

```diff
@@ -0,0 +1,269 @@
+    var matchingtype3 = Model.MatchingCompetenciesType3;
+    var differingtype3 = Model.DifferingCompetenciesType3;
+    var matchingcertificates = Model.MatchingCertificates;
+    var differingcertificates = Model.DifferingCertificates;
+}
+    <section role="contentinfo">
+        <div class="row mrgn-tp-md">
+            <button class="btn btn-primary mrgn-lft-md" type="button">@Localizer["Print PDF"]</button>
+            <button class="btn btn-primary pull-right mrgn-rght-md" type="button">@Localizer["Save PDF"]</button>
+        </div>
+
+            <div class="row text-center mrgn-bttm-lg">
+                <h2><b>@Localizer["h2Results"]</b></h2>
+            </div>
+            <table class="table table-bordered table-responsive" width="100%" border="0" cellspacing="0" cellpadding="0">
+                <tbody>
+                    <tr>
+                        <td class="bg-success" width="50%"><b>@Localizer["Current Position"]:</b> @current.JobGroupCode @current.JobGroupLevelValue - @(isEnglish?current.JobTitleEng:current.JobTitleFre)</td>
+                        <td class="bg-info" width="50%"><b>@Localizer["Selected Position"]:</b> @objective.JobGroupCode @objective.JobGroupLevelValue - @(isEnglish?objective.JobTitleEng:objective.JobTitleFre)</td>
+                    </tr>
+                </tbody>
+            </table>
+
+            <h3>@Localizer["MATCHING COMPETENCIES"]</h3>
+
+            <table class="table table-responsive table-bordered">
+
+                <tr>
+                    <th style="background-color:#f2f2f2" width="50%">@Localizer["CERTIFICATIONS"]</th>
+                    <th style="background-color:#f2f2f2" width="50%">@Localizer["TECHNICAL ELEMENTS"]</th>
+                </tr>
+
+                @if (matchingtype2.Length >= matchingcertificates.Length)
+                {
+                    @for (int i = 0; i < matchingtype2.Length; i++)
+                    {
+                        if (i < matchingcertificates.Length)
+                        {
+                            <tr>
+                                <td width="50%">@(isEnglish?matchingcertificates[i].NameEng:matchingcertificates[i].NameFre)</td>
+                                <td width="50%">@(isEnglish?matchingtype2[i].CompetencyNameEng:matchingtype2[i].CompetencyNameFre)</td>
+                            </tr>
+                        }
+                        else
+                        {
+                            <tr>
+                                <td width="50%">@null</td>
+                                <td width="50%">@(isEnglish?matchingtype2[i].CompetencyNameEng:matchingtype2[i].CompetencyNameFre)</td>
+                            </tr>
+                        }
+                    }
+                }
+                else
+                {
+                    @for (int i = 0; i < matchingcertificates.Length; i++)
+                    {
+                        if (i < matchingtype2.Length)
```

Comment:
This if statement should be on the second column since the first colum is the same in the if and else